sdk/python/kfp/dsl/__init__.py

```diff
@@ -266,6 +266,7 @@ def my_pipeline():
     from kfp.dsl.pipeline_channel import OneOf
     from kfp.dsl.pipeline_context import pipeline
     from kfp.dsl.pipeline_task import PipelineTask
+    from kfp.dsl.pipeline_config import PipelineConfig
```

Comment:
The isort CI check is failing: https://github.com/kubeflow/pipelines/actions/runs/10433635473/job/28895596111?pr=11112
The `isort` tool checks that imports are organized in a specific order, such as standard library imports first, followed by third-party libraries, and then local imports.
Running this locally should fix it: `isort --profile google sdk/python/kfp/dsl/__init__.py`